src/components/Text.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+export default function Text({ 
+  text = "",
+  italic = false, 
+  bold = false, 
+  underline = false, 
+  size = 'text-base', // uses tailwind utility class to set 1rem
+  color = 'text-black', 
+}) { 
```

Comment:
Solved by resolving the above comment, the Text's props are not explicitly typed. We should strive to have total Type Coverage for projects that utilize TS.